e2e-tests/batch-mode-tests.sh

```diff
@@ -0,0 +1,101 @@
+print_prefix="E2E: BATCH MODE TEST:"
+echo "START OF BATCH MODE TEST"
+echo "${print_prefix} STARTING OPENMRS SERVER"
+docker-compose -f openmrs-compose.yaml up -d
+statuscode=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/rest/v1/visittype)
```

Comment:
I'd suggest this in other places too:

```suggestion
openmrs_start_wait_time=0
statuscode=$(curl -o /dev/null --head -w "%{http_code}\n" -L -X GET -u admin:Admin123 --connect-timeout 5 --max-timeout 20 http://localhost:8099/openmrs/ws/rest/v1/visittype 2>/dev/null)
```

Reply:
@ibacher for some reasons running  ```curl -o /dev/null --head -w "%{http_code}\n" -L -X GET -u admin:Admin123 --connect-timeout 5 --max-timeout 20 http://localhost:8099/openmrs/ws/rest/v1/visittype 2>/dev/null``` is giving no response....

Reply:
I mean the status code value is also null